### PR TITLE
Give IE9 the no-js experience

### PR DESCRIPTION
**Pull request by anselmbradford**
Fixes https://github.com/cfpb/cfgov-refresh/issues/3687

## Removals

- `common.ie.js` IE9-specific JS.
- IE scripts build task.
- `class-list.js` polyfill.

## Changes

- Merges IE8 and IE9 style build tasks.
- Increment conditional comments by one to ignore JS in IE9.
- Drop `script.readyState` check in dynamic JS loader (for IE9).

## Testing

1. `gulp clean && gulp build`.
2. Check site in IE9 and IE8.

### CODE REVIEW

File: cfgov/unprocessed/js/modules/util/js-loader.js [hunk 1]
```diff
@@ -12,29 +12,12 @@ function loadScript( url, callback ) {
...
-        script.onreadystatechange = null;
-        if ( callback ) {
-          return callback();
-        }
-      }
-      return null;
-    };
-  // Other browsers.
-  } else {
-    script.onload = function() {
```
Comment (anselmbradford) [inline cfgov/unprocessed/js/modules/util/js-loader.js]: This is the support table for script.onload http://kangax.github.io/jstests/script-element-onload-attribute-support/

File: gulp/tasks/scripts.js [hunk 2]
```diff
@@ -212,7 +200,6 @@ function scriptsOAH() {
 gulp.task( 'scripts:polyfill', scriptsPolyfill );
 gulp.task( 'scripts:modern', scriptsModern );
 gulp.task( 'scripts:oah', scriptsOAH );
-gulp.task( 'scripts:ie', scriptsIE );
```
[the remark this answers is not in the document]
Comment (jimmynotjim) [reply gulp/tasks/scripts.js]: Came to report the same issue.

[the remark this answers is not in the document]
Comment (anselmbradford) [reply gulp/tasks/scripts.js]: Argh, mistake while resolving conflicts twice after the gulp 4 PR. Fixed

[the remark this answers is not in the document]
Comment (Scotchester) [reply gulp/tasks/scripts.js]: @jimmynotjim First! :trollface:

File: cfgov/jinja2/v1/_layouts/base.html [hunk 3]
```diff
@@ -108,8 +108,7 @@
 #}
 
 {% block css %}
-<!--[if lt IE 9]><link rel="stylesheet" href="{{ static('css/main.ie8.css') }}"><![endif]-->
-<!--[if IE 9]><link rel="stylesheet" href="{{ static('css/main.ie9.css') }}"><![endif]-->
```
Comment (Scotchester) [inline cfgov/jinja2/v1/_layouts/base.html]: I'm not sure we want to do this. I haven't gone looking for anything in particular, but IE8 and IE9 may have different issues with CSS. Let's keep the scope of this PR to JavaScript.

Comment (Scotchester) [reply cfgov/jinja2/v1/_layouts/base.html]: Or at least give me a little time tomorrow to research it further.

File: gulp/tasks/styles.js [hunk 4]
```diff
@@ -43,13 +43,13 @@ function stylesModern() {
 }
 
 /**
- * Process legacy CSS for IE9 only.
+ * Process legacy CSS for IE9 and below.
  * @returns {PassThrough} A source stream.
  */
-function stylesIE9() {
+function stylesIE() {
```
Comment (jimmynotjim) [inline gulp/tasks/styles.js]: Should we name this `stylesLegacyIE` so it's clear it doesn't do anything to IE 10 or 11?

File: cfgov/jinja2/v1/_layouts/base.html (hunk 3 above)
Comment (anselmbradford) [reply cfgov/jinja2/v1/_layouts/base.html]: Differences likely would be fallbacks stacked on top of each other, in which case one file would likely be preferable.

File: gulp/tasks/styles.js (hunk 4 above)
Comment (anselmbradford) [reply gulp/tasks/styles.js]: Technically 10/11 are legacy too 😁

Comment (jimmynotjim) [reply gulp/tasks/styles.js]: Well shoot...

**Comment by anselmbradford** [comment]
@Scotchester @jimmynotjim how's this look?

**Comment by Scotchester** [comment]
Approval contingent on rebasing before merging to get rid of the dozen "Update branch" merge commits ;)

**Comment by Scotchester** [comment]
Works for me after the fancy rebase. Thanks for indulging my OCD 🙂

**Pull request merged**